src/libostree/ostree-repo.c

```diff
@@ -3526,7 +3526,7 @@ ostree_repo_read_commit (OstreeRepo   *self,
 gboolean
 ostree_repo_pull (OstreeRepo               *self,
                   const char               *remote_name,
-                  char                    **refs_to_fetch,
+                  const char              **refs_to_fetch,
```

Comment:
(And we just have to eat the pain of casts between `char**` and `const char * const*`).